### ISSUE TITLE
Showing rules

### ISSUE THREAD

**Issue by ashmonger** [fusioninventory/fusioninventory-for-glpi#1824]
Hello

When using the NetDiscovery Module, it would be nice to know which rule prevent the item to be imported:

> 2016-05-24 15:28:17	En cours d'exécution	Import refusé [mac]:28:10:7b:xx:xx:xx, [ip]:172.17.XX.XX, [name]:TheName, [entities_id]:0, [itemtype]:NetworkEquipment

The item seems to be effectively detected, and nicely, but I don't know what to tweak to make it in the inventory.

**Comment by ltbplinux** [comment]
Yes its a great idea, i'm having problems to understand how rules works when i'm trying to do a netdiscover task. With the default rules, the discovered devices hadn't been imported to unmanaged devices. It refuses each new netdiscover.

**Comment by ltbplinux** [comment]
if i create a rule with some criterias i have duplicate entries each time that the netdiscover task runs
